Sources/Anthropic/Private/Network/Endpoint.swift

```diff
@@ -27,12 +27,12 @@ protocol Endpoint {
 
 extension Endpoint {
 
-   private func urlComponents(
+   func urlComponents(
```

Comment:
@jamesrochabrun This is one of the parts of your core lib that I touched. I am re-using this `urlComponents` helper from `Endpoint+AIProxy.swift`